js/apps/admin-ui/src/clients/advanced/OpenIdVerifiableCredentials.tsx

```diff
@@ -0,0 +1,92 @@
+import {
+  FormGroup,
+  Switch,
+  Text,
+  Button,
+  ActionGroup,
+} from "@patternfly/react-core";
+import { useTranslation } from "react-i18next";
+import { HelpItem } from "@keycloak/keycloak-ui-shared";
+import { FormAccess } from "../../components/form/FormAccess";
+import { Controller, useFormContext } from "react-hook-form";
+import { convertAttributeNameToForm } from "../../util";
+import { FormFields, SaveOptions } from "../ClientDetails";
+import ClientRepresentation from "libs/keycloak-admin-client/lib/defs/clientRepresentation";
+
+type OpenIdVerifiableCredentialsProps = {
+  client: ClientRepresentation;
+  save: (options?: SaveOptions) => void;
+  reset: () => void;
+  hasConfigureAccess?: boolean;
+};
+
+export const OpenIdVerifiableCredentials = ({
+  client,
+  save,
+  reset,
+  hasConfigureAccess,
+}: OpenIdVerifiableCredentialsProps) => {
+  const { t } = useTranslation();
+  const { control } = useFormContext();
+
+  return (
+    <>
+      <Text className="pf-v5-u-pb-lg">
+        {t("openIdVerifiableCredentialsHelp")}
+      </Text>
+      <FormAccess
+        role="manage-clients"
+        fineGrainedAccess={hasConfigureAccess}
+        isHorizontal
+      >
+        <FormGroup
+          label={t("oid4vciEnabled")}
+          fieldId="oid4vciEnabled"
+          hasNoPaddingTop
+          labelIcon={
+            <HelpItem
+              helpText={t("oid4vciEnabledHelp")}
+              fieldLabelId="oid4vciEnabled"
+            />
+          }
+        >
+          <Controller
+            name={convertAttributeNameToForm<FormFields>(
+              "attributes.oid4vci.enabled",
+            )}
+            defaultValue={client.attributes?.["oid4vci.enabled"] || "false"}
+            control={control}
+            render={({ field }) => (
+              <Switch
+                id="oid4vciEnabled-switch"
+                label={t("on")}
+                labelOff={t("off")}
+                isChecked={field.value === "true"}
+                onChange={(_event, value) => field.onChange(value.toString())}
+                aria-label={t("oid4vciEnabled")}
+              />
+            )}
+          />
```

Comment:
You can use the `DefaultSwitchControl` instead:
```suggestion
          <DefaultSwitchControl
              name={convertAttributeNameToForm<FormFields>(
                 "attributes.oid4vci.enabled",
              )}
              label={t("oid4vciEnabled")}
              labelIcon={t("oid4vciEnabledHelp")}
              stringify
            />
```